tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -384,6 +389,11 @@ install_initrd() {
 		if [ "${variant}" == "confidential" ]; then
 			export COCO_GUEST_COMPONENTS_TARBALL="$(get_coco_guest_components_tarball_path)"
 			export PAUSE_IMAGE_TARBALL="$(get_pause_image_tarball_path)"
+			# GO_VERSION should be exported to install the package in ubuntu-rootfs-osbuilder
+			# This is necessary for installing opa from the source for s390x and ppc64le
+			if [ "${AGENT_POLICY}" == "yes" ] && [ "${ARCH}" == "s390x" -o "${ARCH}" == "ppc64le" ]; then
```

Comment:
Hi @BbolroC ! I've some doubts that I won't be able to study first today. However, one thing that I can ask:

Is this `if` block in the path of s390x builds? I figured that `variant==confidential` when you build the `rootfs-image-confidential` asset. The .github/workflows/build-kata-static-tarball-s390x.yaml workflow seems to build both `rootfs-image` and `rootfs-initrd` only.

Reply:
Yes, you are right. The mentioned code block is not reached by the current workflow. The PR is tested (see https://github.com/BbolroC/kata-containers/actions/runs/8711566342/job/23908856311) and raised based on an assumption that a follow-up PR https://github.com/kata-containers/kata-containers/pull/9494 uses the change of this PR (it includes building 3 new confidential artifacts). Thanks!

Reply:
Cool! thanks for the info!